evap/static/scss/components/_buttons.scss

```diff
@@ -158,7 +158,7 @@ a:not([href]):not(.disabled) {
     .button-open-notebook {
         position: fixed;
         bottom: $footer-height + 20px;
-        z-index: $zindex-fixed;
+        z-index: $zindex-sticky - 1;
```

Comment:
What is the rationale here: Why sticky instead of fixed, why -1? Can we add a comment?

Reply:
```suggestion
        z-index: $zindex-sticky - 1;  /* be behind sticky navbar menus */
```

Reply:
Yup, sounds good :+1: